test/distributed/test_c10d_nccl.py

```diff
@@ -196,36 +197,52 @@ def tearDown(self):
         torch.cuda.device_count() > 0, "GPUs are available, skipping test"
     )
     def test_init_no_gpus(self):
-        store = c10d.FileStore(self.file.name, self.world_size)
+        store = c10d.FileStore(self.file_name, self.world_size)
```

Comment:
Shouldn't this still be self.file.name since this test is not a MultiProcessTestCase?